### DISCUSSION TITLE
Welcome to CIT_CLUB_OFFICIAL_WEB_APP_-FRONTEND Discussions!

### DISCUSSION THREAD

**Discussion by kissingerbrian** [C-I-T-CLUB/CIT_CLUB_OFFICIAL_WEB_APP_-FRONTEND#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other and discuss anything affecting our frontend code. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pollandonderi** [comment]
we shoulld begin as soon as possible, i cant see the discussions if there have been any.

**Reply by kissingerbrian** [reply]
Hi Polland. There is one with project setup

**Reply by kissingerbrian** [reply]
Hey there Polland, Check the Team discussion tab. not repository discusion tab. I have just nocited @sifuna2002  
posted it there not here. Front End team discussion tab.

**Comment by cleo-cyber** [comment]
Hey paullaster you talked about pairing  those using vanilla js with those who are proficient in react.  Also the assigning of pages has not yet been done. I am asking if you can look into it. Thank you

**Comment by kissingerbrian** [comment]
Hi Kadima. Yes I did talk about it. I will have it done by today. Be on the
lookout for it. Also, I may not know how many there are or exactly who.
They need to let me know in time.
RoboticaVision

**Reply by cleo-cyber** [reply]
Sifuna has a list i think of the frontend contributors you can reach out to him then you give us the direction.

**Comment by kissingerbrian** [comment]
Will do after sabbath
RoboticaVision

**Comment by cyrillekey** [comment]
For the registration form do we realy need to collect all those information like programming rating ?. I suggest collecting those in the profile section to make the registration journey as short as possible. People hate filling long forms. I for one do

**Reply by sifunadonnex** [reply]
I think ehhh the details required should be or is from the api for registration from backend

**Comment by cyrillekey** [comment]
That means the registration api requires the design rating and programming rating values plus course and interest by default

**Comment by kissingerbrian** [comment]
Hi Cyrille,
Yes it requires those details.
I think we'll make it seamless in the next version
RoboticaVision
